articles/logic-apps/tutorial-build-schedule-recurring-logic-app-workflow.md

```diff
@@ -185,7 +185,7 @@ Provide these connection details, and choose **Create**.
    |||| 
 
    For more information about these parameters, see 
-   [Calculate a route](https://msdn.microsoft.com/library/ff701717.aspx).
+   [Calculate a route](/bingmaps/rest-services/routes/calculate-a-route).
```

Comment:
Please use this URL: https://docs.microsoft.com/bingmaps/rest-services/routes/calculate-a-route